tensorboard/components/tf_tensorboard/test/tensorboardTests.ts

```diff
@@ -23,11 +27,32 @@ namespace tf_tensorboard {
       });
 
       it('renders injected content', function() {
-        let injected = tensorboard.querySelector('#inject-me');
-        chai.assert.isNotNull(injected);
+        const overview = tensorboard.querySelector('#custom-overview');
+        checkSlottedUnderAncestor(overview, '#content-pane');
+
+        const headerItem1 = tensorboard.querySelector('#custom-header-item1');
+        const headerItem2 = tensorboard.querySelector('#custom-header-item2');
+        checkSlottedUnderAncestor(headerItem1, '.header');
+        checkSlottedUnderAncestor(headerItem2, '.header');
+
+        function checkSlottedUnderAncestor(
```

Comment:
several higher level comments:
1. hoist this up function since it is general utility.
2. `Polymer.dom($0).deepContains($1)` may want to look into this.
3. consider moving new additions in tf_util to testing.ts or tf_util/shadow_dom.ts since we probably want some common utilities for ShadowDom that are missing from `Polymer.dom`.

Reply:
1. Done
2. Thanks, I didn't know Polymer offered this!
3. I'll keep in mind for the future.  Polymer's `deepContains()` addresses the need here, but I imagine we might want our own Shadow utils in the future